upgrade-notes/backend_20240806_094817.md

```diff
@@ -0,0 +1,4 @@
+#### [framework] address, name and some other user data is now nullable - fix ([#3314](https://github.com/shopsys/shopsys/pull/3314))
```

Comment:
I would rather update `upgrade-notes/backend_20240725_061952.md` with something like
```
- see also #project-base-diff for additional fix (https://github.com/shopsys/shopsys/pull/3314)
```

Reply:
and fronted upgrade notes should be created here IMHO

Reply:
and fronted upgrade notes should be created here IMHO